certification/internal/policy/operator/deployable_by_olm_test.go

```diff
@@ -140,6 +139,9 @@ var _ = Describe("DeployableByOLMCheck", func() {
 				Expect(err).ToNot(HaveOccurred())
 				Expect(ok).To(BeTrue())
 			})
+			AfterEach(func() {
+				os.Unsetenv("PFLT_DOCKERCONFIG")
```

Comment:
I don't think there's anything to do here, but I wanted to confirm: This is an unrelated change, right? This doesn't impact the ability to deploy to non-default channels?

Reply:
That's right. While doing the other unit test, I realized a deferred statement might not work. That's why I updated this test as well.